src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1449,6 +1449,90 @@ void valid() {
                 assertAndDump(model, "NIST.valid: confidence must be between 0 and 1", nistSeedlingValidator, true);
             }
         }
+
+        // CompoundJustification must be used only for justifications of argument assertions,
+        // and not for justifications for entities, events, or relation KEs
+        @Nested
+        class RestrictCompoundJustification {
+            @Test
+            void invalid() {
+
+                //test entity
+                final Resource newEntity = makeEntity(model, getEntityUri(), system);
+                addType(newEntity, SeedlingOntology.GeneralAffiliation_APORA);
+                final Resource justification1 = makeTextJustification(model, "source1", 0, 4, system, 1d);
```

Comment:
Same normal and compound justifications are used for newEntity, newRelation, and newEvent. I would recommend just re-using it unless you think that recreating each one has some instructional value

Reply:
No real value, this is essentially three unit tests wrapped into a single test so I treated each one as a isolated test. I'll update to reuse the same normal and compound justifications.

Reply:
Updated to reuse single justification and compound justification Resource.